docs/java-rest/high-level/dataframe/put_data_frame.asciidoc

```diff
@@ -0,0 +1,93 @@
+--
+:api: put-data-frame-transform
+:request: PutDataFrameTransformRequest
+:response: AcknowledgedResponse
+--
+[id="{upid}-{api}"]
+=== Put Data Frame Transform API
+
+The Put Data Frame Transform API is used to create a new {dataframe-job}.
```

Comment:
I am not sure how `{dataframe-job}` is defined in the docs. How do these types of macros work?

Reply:
See https://github.com/elastic/docs/pull/688/files

Reply:
'data frame analytics job' is a mouthful I raised elastic/docs#700